src/wallet.js

```diff
@@ -1161,14 +1159,16 @@ Wallet.prototype.sendMany = function(params, callback) {
       otp: params.otp,
       // The below params are for logging only, and do not impact the API call
       estimatedSize: transaction.estimatedSize,
-      feeRate: feeRate,
-      fee: fee,
       buildParams: preservedBuildParams
     });
   })
   .then(function(result) {
-    result.fee = fee;
-    result.feeRate = feeRate;
+    const tx = bitcoin.Transaction.fromHex(result.tx);
+    const inputsSum = _.sumBy(unspentsUsed, (entry) => { return _.get(entry, 'value', 0); });
```

Comment:
side note: if you have a function body where the only statement is a `return`, you can write `(x) => y` instead of `(x) => { return y; }`

However in this case you can just use `_.sumBy(unspentsUsed, 'value');`

Instead of `unspentsUsed` we should use `tx.ins` here since the type is clearer and it the potential for error is lower.

Both `_.get(entry, 'value', 0)` and `_.sumBy()` are not ideal since we actually want an error when `entry.value` is `undefined`, and both interpret `undefined` as `0`. But we can trust `tx` to be well-formed in this case.

Reply:
thanks for advice about the syntax.
regarding tx.ins, i commented below as well, but i don't believe tx.ins contains the value property